### ISSUE TITLE
Support ARM64 WSL on ARM64 devices

### ISSUE THREAD

**Issue by nathansoz** [OmniSharp/omnisharp-vscode#3536]
## Issue Description ##
Would like to use x86 VSCode on my SPX as a frontend to dotnet running on WSL (ARM64). When I try to install the extension I get two errors:

```
Starting OmniSharp server at 1/29/2020, 11:49:28 AM
    Target: <snip>

[ERROR] Error: spawn /home/nsosnov/.vscode-server/extensions/ms-vscode.csharp-1.21.9/.omnisharp/1.34.9/run ENOENT
```

And

```
[WARNING]: Processor architecture 'aarch64' is not currently supported by the .NET Core debugger. Debugging will not be available.
```

I thought I have seen issues on github indicating that ARM64 is now supported by the .net core debugger (#2105). There looks to be some manual configuration pointed to, but it isn't clear to me that doing this would actually unblock omnisharp. 

## Steps to Reproduce ##
Use the WSL remote extension on an ARM64 device and attempt to open a c# project. See errors as above.

## Expected Behavior ##
The c# debugger should download and omnisharp should start up properly. 

## Actual Behavior ##
Omnisharp does not start.

### OmniSharp log ###
<details>Starting OmniSharp server at 1/29/2020, 11:49:28 AM
    Target: <snip>

[ERROR] Error: spawn /home/nsosnov/.vscode-server/extensions/ms-vscode.csharp-1.21.9/.omnisharp/1.34.9/run ENOENT</details>

### C# log ###
<details>[WARNING]: Processor architecture 'aarch64' is not currently supported by the .NET Core debugger. Debugging will not be available.</details>

## Environment information ##

**VSCode version**: 1.41.1
**C# Extension**: 1.21.9

<details><summary>Mono Information</summary>
        OmniSharp using built-in mono</details>
<details><summary>Dotnet Information</summary>
.NET Core SDK (reflecting any global.json):
 Version:   3.1.101
 Commit:    b377529961

Runtime Environment:
 OS Name:     ubuntu
 OS Version:  18.04
 OS Platform: Linux
 RID:         ubuntu.18.04-arm64
 Base Path:   /home/nsosnov/dotnet/sdk/3.1.101/

Host (useful for support):
  Version: 3.1.1
  Commit:  a1388f194c

.NET Core SDKs installed:
  3.1.101 [/home/nsosnov/dotnet/sdk]

.NET Core runtimes installed:
  Microsoft.AspNetCore.App 3.1.1 [/home/nsosnov/dotnet/shared/Microsoft.AspNetCore.App]
  Microsoft.NETCore.App 3.1.1 [/home/nsosnov/dotnet/shared/Microsoft.NETCore.App]

To install additional .NET Core runtimes or SDKs:
  https://aka.ms/dotnet-download
</details>
<details><summary>Visual Studio Code Extensions</summary>

|Extension|Author|Version|
|---|---|---|
|csharp|ms-vscode|1.21.9|
|docomment|k--kato|0.1.8|
|dotnet-test-explorer|formulahendry|0.7.1|;

</details>

**Comment by gregg-miskelly** [comment]
I think this is really just a special case for https://github.com/OmniSharp/omnisharp-vscode/issues/2771, so we don't need a separate issue.

**Comment by nathansoz** [comment]
Sounds good.
